libs/core/src/lib/busy-indicator/busy-indicator.component.html

```diff
@@ -1,25 +1,25 @@
 <ng-content></ng-content>
 
 <ng-container *ngIf="loading">
-
     <div
         #busyIndicator
         class="fd-busy-indicator"
-        [ngClass]="{
-        'fd-busy-indicator--l': size === 'l',
-        'fd-busy-indicator--m': size === 'm',
-        'fd-busy-indicator--s': size === 's',
-        'fd-busy-indicator--absolute': busyIndicator?.previousElementSibling
-    }">
+        [class.fd-busy-indicator--l] = "size === 'l'"
+        [class.fd-busy-indicator--m] = "size === 'm'"
+        [class.fd-busy-indicator--s] = "size === 's'"
+        [class.fd-busy-indicator--absolute] = "busyIndicator?.previousElementSibling"
+    >
         <div class="fd-busy-indicator--circle-0"></div>
         <div class="fd-busy-indicator--circle-1"></div>
         <div class="fd-busy-indicator--circle-2"></div>
     </div>
 
-    <div class="fd-busy-indicator__overlay"
-         [class.fd-busy-indicator__overlay--transparent]="!busyIndicator?.previousElementSibling">
-    </div>
+    <span *ngIf="label" class="fd-busy-indicator-extended__label">{{ label }}</span>
```

Comment:
in the styles we say that the `label` can be used only within wrapper class `extended`

![image](https://user-images.githubusercontent.com/3263744/128017328-5140cfab-5915-405b-9f7e-7516dc9bb427.png)

Here we just add label without adding a wrapper class